.gitignore

```diff
@@ -20,4 +20,6 @@ mono_crash.*.json
 # ignore potentially large filetypes
 *.csv
 *.obj
-*.ogg
\ No newline at end of file
+*.ogg
+*.png
+*.jpg
```

Comment:
I think we discussed this before, but don't we want include these artifacts to ensure the reproducibility of the project?

Reply:
They are not included for the same reason as the `.csv`, `.ogg`, and `.obj` files, which is that they can be large and take up too much space in the repo.

Reply:
Are they all large?  I think we need to find some solution to this.  Can you post some information about file sizes?

Reply:
`.csv`
- `points.csv` is 222KB
- `tank10.csv` is 223MB
- `tworoom10msht.csv` is 28MB

`.obj`
- `GeigerCounter.obj` is 3MB
- `TwoRoomModel.obj` is 43MB

`.ogg`
- `geigercounter.ogg` is 6 KB

`.jpg`
- they range from 0.4MB to 2MB, and there are 4 `.jpg` used for each texture. 

Another thing to consider is that the `.git` folder will become very large if there are ever any changes made to these large binary files.

Reply:
Yeah... a little larger than we'd like and I don't want to get into git large file stuff...

That said, we would like to have an easy way for someone else to replicate this. We can host these files somewhere and have a step to download and unpack them into the right place.

We may also want to think about how we separate them out.  For example, I could imagine building a library of textures (jpg) and sounds (ogg), but the geometry (obj) and rad data (csv) are unique to a particular case.